README.md

```diff
@@ -144,6 +164,82 @@ Documentation for the flow package can be accessed using standard godoc tool, e.
 godoc github.com/trustmaster/goflow
 ```
 
+## Runtime registration example
```

Comment:
`Runtime.Register()` should be called only once when a runtime is started to let Flowhub know that it is up and available. For example, it could be added like this to the "Basic Server for Flowhub" example:

```go
runtime.Init()
runtime.Register(runtimeDetails)
```

or

```go
runtime.Init()
runtime.RegisterJSON("flowhub.json")
```